srcpkgs/perl-Class-Data-Inheritable/template

```diff
@@ -1,17 +1,17 @@
-# Template build file for 'perl-Class-Data-Inheritable'.
+# Template file for 'perl-Class-Data-Inheritable'
 pkgname=perl-Class-Data-Inheritable
 version=0.08
-revision=1
+revision=2
+noarch=yes
 wrksrc="${pkgname#*-}-${version}"
 build_style=perl-module
 hostmakedepends="perl"
 makedepends="perl"
-checkdepends="perl-Test-Pod perl-Test-Pod-Coverage"
 depends="perl"
-noarch=yes
+checkdepends="perl-Test-Pod perl-Test-Pod-Coverage"
 short_desc='Class::Data::Inheritable - Inheritable, overridable class data'
 maintainer="Enguerrand de Rochefort <voidlinux@rochefort.de>"
-homepage="https://metacpan.org/release/Class-Data-Inheritable"
 license="GPL-1, Artistic"
```

Comment:
license="Artistic-1.0-Perl, GPL-1.0-or-later"

Reply:
ok, i''ll change that license everywhere